hw/ip/alert_handler/doc/_index.md

```diff
@@ -128,13 +128,13 @@ configuration, and hence these parameters are placed into a package as
 "localparams". The parameterization rules are explained in more detail in the
 architectural description.
 
-Localparam     | Default (Max)         | Description
----------------|-----------------------|---------------
-`NAlerts`      | 8 (248)               | Number of alert instances. Maximum number bounded by LFSR implementation that generates ping timing.
-`EscCntWidth`  | 32 (32)               | Width of the escalation counters in bit.
-`AccuCntWidth` | 16 (32)               | Width of the alert accumulation counters in bit.
-`AsyncOn`      | '0 (2^`NAlerts`-1)    | This is a bit array specifying whether a certain alert sender / receiver pair goes across an asynchronous boundary or not.
-`LfsrSeed`     | '1 (2^31-1)           | Seed for the LFSR timer, must be nonzero.
+Localparam     | Default (Max)         | Top Earlgrey | Description
+---------------|-----------------------|--------------|---------------
+`NAlerts`      | 8 (248)               | 1            | Number of alert instances. Maximum number bounded by LFSR implementation that generates ping timing.
```

Comment:
For the time being this is probably ok, but eventually we should discuss how we could infer these from the configuration.

Reply:
Oh sure, I think this should eventually be generated too, but I felt that was too much for this PR.